addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/IntegerOverflowScanRuleUnitTest.java

```diff
@@ -91,4 +94,40 @@ public void shouldSkipScanIfStopped() throws Exception {
         // Then
         assertThat(httpMessagesSent, is(empty()));
     }
+
+    @Test
+    public void shouldCreateAlertsWithPrimeHeaderAsEvidence() throws Exception {
+        String test = "/shouldReportIntegerOverflowIssue/";
+        int INT_MAX = 2147483647;
```

Comment:
Why not just use the Java constant?
`Integer.MAX_VALUE`

Reply:
INT_MAX is from c/c++. Integer overflows may be vulnerability in c/c++ I may be wrong but in other languages it will never be a security issue, even less if it results in an error that is what I think the rule checks.

Reply:
Oh okay, how I get why you went that way.